hs_core/search_indexes.py

```diff
@@ -1,5 +1,9 @@
 """Define search indexes for hs_core module."""
 
+# NOTE: this has been optimized for the current and future discovery pages.
+# Features that are not used have been commented out temporarily
+# and marked as # unused # for future deletion once discovery is revised.
```

Comment:
ok, I see your note now, disregard my previous comment about deleting the commented out code